src/cclib/parser/orcaparser.py

```diff
@@ -551,6 +551,55 @@ def extract(self, inputfile, line):
                 self.gbasis.append(gbasis_tmp[bas_atname])
             del self.tmp_atnames
 
+        """
+        --------------------------
```

Comment:
Orca has a lot of filler in the thermochemistry output and breaks it up into a bunch of sections (some of this filler changed upon release of version 4; doesn't affect my parsing). It would take >100 lines, so I don't think it is worthwhile to include a fragment. If you still think it is worthwhile, I can go ahead and add a fragment.